main.cpp

```diff
@@ -518,6 +443,37 @@ Part 1d - Step 4: Assignment
         Brightness (double)
         width in cm (int)
         height in cm (int)
+
+    Gears:
+        number of gears (int)
+        brand (std::string)
+        type (std::string)
+        condition (char)
+        need maintenance? (bool)
+    Frame:
+        weight (int)
+        type (std::string)
+        material (std::string)
+        brand (std::string)
+        need maintenance? (bool)
+    Wheel:
+        circumference (float)
+        number of spokes (int)
+        material (std::string)
+        brand (std::string)
+        need maintenance? (bool)
+    Brakes:
+        type (std::string)
+        brand (std::string)
+        material (std::string)
+        amount of pad left (float)
+        need maintenance? (bool)
+    Handlebars:
+        type (std::string)
+        material (std::string)
+        width (float)
+        brand (std::string)
+        need maintenance? (bool)
```

Comment:
Delete this.  I'm not even sure why you wrote this here, as the instructions did not specify to.
- [ ] revise